apps/ui/lib/components/LinkModal/LinkModal.tsx

```diff
@@ -190,20 +185,18 @@ export const LinkModal: React.FunctionComponent<Props> = ({
 				{cards[0].name || cards[0].slug}
 			</Txt>
 			{!linkVerb && (
-				<HideableBox isHidden={!selectedTarget || linkTypeTargets.length === 1}>
```

Comment:
@LucianBuzzo @karaxuna Not sure what the point of this `HideableBox` is, but please correct me if we need to keep it. The link modal works as I expect it to after removing it and only using the wrapped `Select` component.

Reply:
@joshbwlng IIRC the hideable box exists so that you can omit the `Select` if there is only 1 possible option for the relationship.

Reply:
@LucianBuzzo @karaxuna I was confused because while it does *kind of* hide the box, its actually still there if you click it:
![HideableBox](https://user-images.githubusercontent.com/45343541/179649392-49569119-fa7d-416e-8d0c-6e4c025351c7.png)

Reply:
And I actually think it feels fine to have a select with only one option - a consistent feel as the options are filtered down when a target type and contract is selected.

Reply:
If everyone agrees I think we can merge this. Let me know if you think there's a better option though.

Reply:
Ok sure, lets merge it